src/main/java/uk/gov/companieshouse/kafka_email/model/EmailSend.java

```diff
@@ -0,0 +1,8 @@
+package uk.gov.companieshouse.kafka_email.model;
+
+import email.email_send;
+
+/**
+ * Extends {@link email_send} Avro schema from kafka-models, adhering to Java naming conventions.
+ */
+public class EmailSend extends email_send {}
```

Comment:
Name is bit wired. Sounds more like verb less like noun.

Reply:
If I had to choose repeating bad name for consistency vs fixing name I'd not think twice.

But the whole API is does not hide the fact that there is messaging system underneath so expectation are kind of low already.

Also the fact that consuming services needs to create EmailService is bad sign. I'd expect this to be just `emailService.send('foo@bar', data)`  for example executed from the controller.

Reply:
Fixed (along with major refactor) as discussed.